arch/arm/kernel/traps-v7m.c

```diff
@@ -123,7 +123,7 @@ void traps_v7m_init(void){
 	writel(readl(&NVIC->system_handler_csr) | USGFAULTENA | BUSFAULTENA,
 			&NVIC->system_handler_csr);
 	writel(
-#ifndef CONFIG_ARM_V7M_NO_UNALIGN_TRP
+#if 0 /* commented out until unaligned access in uclibc is fixed */
```

Comment:
Let's do this as follows (not sure about the exact grammar, please check): 

if !defined(CONFIG_ARM_V7M_NO_UNALIGN_TRP) & 0

We want to preserve a reference to CONFIG_ARM_V7M_NO_UNALIGN_TRP in this code.